src/parser/scanner.lex

```diff
@@ -284,6 +285,7 @@ IP_OCTET                    ([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])
 {SNAPSHOT}                  { return TokenType::KW_SNAPSHOT; }
 {SNAPSHOTS}                 { return TokenType::KW_SNAPSHOTS; }
 {FORCE}                     { return TokenType::KW_FORCE; }
+{SCHEMA}                    { return TokenType::KW_SCHEMA; }
```

Comment:
I think we can reuse the keyword TAG and EDGE instead of add new one, and can provide small granular operation.